src/test/java/org/folio/api/BaseIT.java

```diff
@@ -268,4 +280,28 @@ protected MessageHeaders getMessageHeaders(String tenantName, String tenantId) {
     return new MessageHeaders(header);
   }
 
+  @SneakyThrows
+  protected void mockUserTenants() {
+    wireMockServer.stubFor(get(urlEqualTo("/user-tenants?limit=1"))
+      .willReturn(okJson(new JSONObject()
+        .put("totalRecords", 1)
+        .put("userTenants", new JSONArray()
+          .put(new JSONObject()
+            .put("centralTenantId", CENTRAL_TENANT_ID)
+            .put("consortiumId", CONSORTIUM_ID)
+            .put("userId", UUID.randomUUID().toString())
+            .put("tenantId", UUID.randomUUID().toString())))
+        .toString())));
+  }
+
+  @SneakyThrows
+  protected void mockConsortiaTenants() {
+    wireMockServer.stubFor(get(urlEqualTo(format("/consortia/%s/tenants", CONSORTIUM_ID)))
+      .willReturn(jsonResponse(new JSONObject()
+        .put("tenants", new JSONArray(Set.of(
+          new JSONObject().put("id", "consortium").put("isCentral", "true"),
+          new JSONObject().put("id", "university").put("isCentral", "false"),
+          new JSONObject().put("id", "college").put("isCentral", "false")
+        ))).toString(), HttpStatus.SC_OK)));
+  }
```

Comment:
Are we sure these methods belong in this class? Should they really be accessible to all descendants of `BaseIT`?

Reply:
done